src/corecel/sys/ScopedProfiling.hip.cc

```diff
@@ -32,19 +34,21 @@ bool ScopedProfiling::enable_profiling()
     static bool const result = [] {
         if (!celeritas::getenv("CELER_ENABLE_PROFILING").empty())
         {
-            if (CELERITAS_HAVE_ROCTX)
+            if (!celeritas::device())
```

Comment:
I feel like this could still be useful to allow the use of ScopedProfiling in a device build with the environment variable disabling the device for CPU profiling.

Reply:
Oh, so the profilers will still gather useful CPU only data? That would indeed be useful. I'll revert that change.

Reply:
It doesn't collect that much data, for purely CPU profiling it'd be better to use another profiler, but it will still show the scope on the timeline, so it's possible to get the time spent in different functions.